frontend/app/services/redis-service.server.ts

```diff
@@ -59,14 +52,16 @@ export function getRedisService() {
      */
     set: async (key: string, value: unknown, options?: SetOptions) => {
       const redisClient = await getRedisClient();
-      return redisClient.set(key, JSON.stringify(value), options);
+      return await redisClient.set(key, JSON.stringify(value), options);
```

Comment:
No need to `await` at the return.

Reply:
Weird that vscode only warns me about this sometimes.